pkg/filters/type.go

```diff
@@ -10,8 +10,8 @@ var (
 )
 
 type Filters struct {
-	FieldSelector string `json:"field-selector" pflag:",Specifies the Field selector"`
-	SortBy        string `json:"sort-by" pflag:",Specifies which field to sort results "`
+	FieldSelector string `json:"fieldSelector" pflag:",Specifies the Field selector"`
+	SortBy        string `json:"sortBy" pflag:",Specifies which field to sort results "`
 	// TODO: Support paginated queries
```

Comment:
With this change , there are docs references as well which we would need to change . 
Can you check all references of field-selector and sort-by in go files and replace them.
eg : in get/launch_plan.go

If you do make gendocs , it will take care to fix the .rst files aswell.

Reply:
Fixed